### PR TITLE
[cmake] Standardize CMake code

### PR DESCRIPTION
**Pull request by bellenot**
Remove special cases for Windows and make it build and run in both builtin and standalone mode

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -56,11 +56,7 @@ if(MSVC)
     # ROOT_BINDIR is set by ROOTConfig.cmake
     get_filename_component(realfp "${ROOT_BINDIR}/../" ABSOLUTE)
     file(TO_CMAKE_PATH ${realfp} ROOTSYS)
-    # FIXME: Get from root-config.bat --tutdir
```
Comment (hahnjo) [inline CMakeLists.txt:59]: Don't we want to keep this as a reminder?

Comment (bellenot) [reply CMakeLists.txt:59]: I tried, but `root-config.bat --tutdir` returns just `tutorials` in buit-in mode on Windows...

Comment (hahnjo) [reply CMakeLists.txt:59]: But shouldn't that be fixed instead? Does it work with the stand-alone build?

Comment (bellenot) [reply CMakeLists.txt:59]: Yes, it works in stand-alone. I'll check what's different between Linux and Windows...

Comment (bellenot) [reply CMakeLists.txt:59]: But it shouldn't really matter anyway (it's  semantically the same)

Comment (bellenot) [reply CMakeLists.txt:59]: @hahnjo I'm fixing `root-config.bat.in` right now. Will fix roottest afterwards

**Comment by bellenot** [comment]
closing this PR, will make a new one when `root-config.bat` is fixed and verified that everything is fine with both builtin and standalone roottest

**Pull request closed without merging**